services/otel-gateway/tests.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os
+import time
+from pathlib import Path
+
+import generate
+import pytest
+
+trace_file = Path("exported/traces.json")
+metric_file = Path("exported/metrics.json")
+
+
+@pytest.fixture(autouse=True)
+def clear_files():
+    # ensure test server has stasted up
```

Comment:
Why has this been commented out, is it still needed?

Reply:
left over debugging, will remove